unit/json/json_parser.cpp

```diff
@@ -84,17 +84,21 @@ SCENARIO("Loading JSON files")
       }
     }
   }
-  GIVEN("A JSON file containing a hexadecimal Unicode character")
+  GIVEN("A JSON file containing hexadecimal Unicode characters")
```

Comment:
:pick: The use of "characters" here is a bit confusing. I would talk about symbols (or code points) for unicode and only use characters for the c++ `char` type :hamburger: